VPLAY-11442 : [Update UVE Doc] Ability to configure + / - av sync thr…

Gnanesha opened the pull request:
…eshold values for MonitorAV.

Reason for change: added documentation for monitorAV sync
Test Procedure: No Test required
Priority: P1

Commit:
VPLAY-11442 : [Update UVE Doc] Ability to configure + / - av sync threshold values for MonitorAV.

Reason for change: added documentation for monitorAV sync
Test Procedure: No Test required
Priority: P1

Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

Commit:
VPLAY-11442 : [Update UVE Doc] Ability to configure + / - av sync threshold values for MonitorAV.

Reason for change: added documentation for monitorAV sync
Test Procedure: No Test required
Priority: P1

Signed-off-by: Gnanesha <Gnanesha_Thimmegowda@comcast.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/aamp?pullRequest=588) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/aamp?pullRequest=588) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: pstroffolino<br/>:x: Gnanesha<br/><hr/>**Gnanesha** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/aamp?pullRequest=588) it.</sub>

pstroffolino committed:
Merge branch 'dev_sprint_25_2' into feature/VPLAY-11442

pstroffolino reviewed (changes requested):
Documentation update needs improvement - as described, use and semantics are not clear at all.

One of these is threshold for audio leading (before) video to be considered unacceptable.

One of these is threshold for audio lagging (late) video to be considered unacceptable.

Just saying "positive" or "negative" doesn't tell me anything as a user what to fill out or what it means.

Also confusing because a user might try and fill in an actual negative number - both values are (per my understanding) non-negative configurations.

Gnanesha committed:
VPLAY-11442 : [Update UVE Doc] Ability to configure + / - av sync threshold values for MonitorAV.

Reason for change: added documentation for monitorAV sync
Test Procedure: No Test required
Priority: P1
